sdk/communication/azure-communication-common/src/test/java/com/azure/communication/common/CommunicationIdentifierTests.java

```diff
@@ -14,27 +14,27 @@
 
     @Test
     public void equalityOnlyTestIdIfPresentOnBothSide() {
```

Comment:
`equalityOnlyTestRawIdIfPresentOnBothSide` (`Id` ➡ `RawId`). I should have done this in C# as well